docs/user-guide/src/bmo/live-iso.md

```diff
@@ -0,0 +1,40 @@
+# Live-ISO
+
+The live-iso API in Metal3 allows to boot BaremetalHost with live-iso image instead of using IPA deploy ramdisk.
+
+Why we need it?
+
+In some circumstances, i.e to reduce boot time for ephemeral workloads, it may be possible to boot an iso
+and not deploy any image to disk (saving the time to write the image and reboot). Since this alternative
+is distributed as a live-iso, leveraging the existing toolchains like [fedora-coreos installer](https://docs.fedoraproject.org/en-US/fedora-coreos/bare-metal/) might be desirable. On top that, Ironic already exposes an [API](https://docs.openstack.org/ironic/latest/admin/ramdisk-boot.html) to support this.
+
+How to use it?
+
+Here is an example with a BaremetalHost CRD, where DiskFormat spec indicates that instead
+of deploying an image to disk, a live-iso image will be booted:
+
+```yaml
+spec:
+image:
+    url: http://1.2.3.4/image.iso
+    format: live-iso
+online: true
+```
+
+**Note**  `rootDeviceHints` and `userData` will not be used
+since the image won't be written to disk, and Ironic doesn't
+support passing user-data in addition to the iso attachment at present.
+
+There are two requirement to enable this feature on Ironic side:
+
+1. Please make sure that Ironic is configured to use the [ramdisk
+   deploy interface](https://docs.openstack.org/ironic/latest/admin/ramdisk-boot.html)
+
+```yaml
+enabled_deploy_interfaces = direct,ramdisk
+```
+
+2. Also, you need to set the provided ISO URL into the
+   [instance_info boot_iso field](https://docs.openstack.org/ironic/latest/admin/drivers/redfish.html#virtual-media-ramdisk)
```

Comment:
If I understand correctly, this is what BMO will do for you behind the scenes, not something that the user has to do, so I would skip it. Or alternatively reword it so that it is clear that this is something BMO does and you don't have to think about it.